test/test_data/annotated_assignment_with_parens.py

```diff
@@ -0,0 +1 @@
+(paren): int = 1 + 2
```

Comment:
This will probably fail as well. The correct AST for this only contains the name itself, but we generate the tuple.